meson.build

```diff
@@ -0,0 +1,26 @@
+project(
```

Comment:
It might be a good idea to set `meson_version: '>=0.64.0'` or something as a kwarg to the project() function to specify the minimum version of Meson you require. If you do, then Meson can warn you when you accidentally use features that are too new, and will error out if people still have an old version installed that won't work.